installer/targets/git-hooks/git_hooks/pre-commit.d/author

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+AUTHORINFO=$(git var GIT_AUTHOR_IDENT) || return 1
+NAME=$(printf '%s\n' "${AUTHORINFO}" | sed -n 's/^\(.*\) <.*$/\1/p')
+EMAIL=$(printf '%s\n' "${AUTHORINFO}" | sed -n 's/^.* <\(.*\)> .*$/\1/p')
+
+echo "I'm going to commit with \"$NAME <$EMAIL>\""
+if [[ "${NAME}" == "amigo" ]]
+then
+    echo "Please commit with a username, run \"git su <name>\" to switch users"
+    return 1
+fi
+
+if [[ "$ROBOT_REAL" == "true" ]]
+then
+    # prompt for conformation
+    exec < /dev/tty
+    read -p "Are you sure? " -n 1 -r
+    exec <&-
+    echo    # (optional) move to a new line
+    if [[ ! $REPLY =~ ^[Yy]$ ]]
+    then
+        echo "You can change the user by running \"git su <name>\""
+        exit 1
```

Comment:
Any reason for using 'exit' here and 'return' in the other check?

Reply:
Eeehm, nope.

Reply:
So, please use the same. Up to you which is the best.